test/support/pages/SettingsPage.js

```diff
@@ -7,7 +7,7 @@ define(function (require) {
   var Promise = require('bluebird');
   var Common = require('./Common');
 
-  var defaultTimeout = 60000;
+  var defaultTimeout = 90000;
```

Comment:
What are we waiting over 60 seconds for?  I can't think of anything in our tests that should take more than about 15 or 20 seconds to load.